### ISSUE TITLE
Wrong focus after closing "New calendar"

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/calendar#4127]
### Steps to reproduce

1. Open the app
2. Attempt to create a new calendar
3. Close the menu again

### Expected behavior

Focus should be moved back to where it was before – in this case to the "New event" button.

### Actual behaviour

Closing "New event" popover moves focus to the start of the page

### Calendar app version

3.x

**Comment by st3iny** [comment]
Can be generalized even more: Focus is not moved to previous element after closing popovers and modal.
